configure

```diff
@@ -3764,16 +3764,29 @@ return f (e, argv, 0) != argv[0]  ||  f (e, argv, 1) != argv[1];
   return 0;
 }
 _ACEOF
-for ac_arg in '' -qlanglvl=extc89 -qlanglvl=ansi -std \
-	-Ae "-Aa -D_HPUX_SOURCE" "-Xc -D__EXTENSIONS__"
-do
-  CC="$ac_save_CC $ac_arg"
-  if ac_fn_c_try_compile "$LINENO"; then :
-  ac_cv_prog_cc_c89=$ac_arg
+if [ "$OMR_ENV_OPENXL" == "1" ]; then
```

Comment:
This file should be derived from `configure.ac` (which is not modified by this change).
How would `OMR_ENV_OPENXL` be defined? (Hint: it's ***_not_*** from `OmrDetectSystemInformation.cmake`).

Reply:
I'm not sure if this is the solution you have in mind, but I opted to replace the check on `OMR_ENV_OPENXL` with the following condition: 
```
if [[ "$CC" =~ 'ibm-clang' ]] && [[ "$CXX" =~ 'ibm-clang' ]]; then
```
since `CC` and `CXX` are both used in `configure.ac`, and when OpenXL being used, they should be set to the following:
``
* C Compiler:     Version 17.0.6 (at /opt/IBM/openxlC/17.1.2/bin/ibm-clang_r)
* C++ Compiler:   Version 17.0.6 (at /opt/IBM/openxlC/17.1.2/bin/ibm-clang++_r)
```
Is this a viable alternative?

Reply:
watch out for possible conflict with zOS using wyvern compiler:
1) do they have the same names?
2) if yes, can they use whatever under that condition?

Reply:
The zOS installation of OpenXL does indeed use the same name. And I don't think there's anything under that condition that's incompatible with zOS, but I will test that out before we proceed just to make sure.